internal/pkg/manifest/lb_web_svc.go

```diff
@@ -46,6 +46,9 @@ type LoadBalancedWebService struct {
 
 // LoadBalancedWebServiceConfig holds the configuration for a load balanced web service.
 type LoadBalancedWebServiceConfig struct {
+	// Fields that are used while marshaling the template for additional clarifications,
+	// but don't correspond to a field in the manifests.
+	AppDomain   *string
```

Comment:
nit: can we move this to the bottom separated by a space:
```suggestion
	Network NetworkConfig `yaml:"network"`

	// Fields that are used while marshaling the template for additional clarifications,
	// but don't correspond to a field in the manifests.
	AppDomain   *string
```